lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -853,6 +853,10 @@ TypeSystemSwiftTypeRef::GetSwiftName(const clang::Decl *clang_decl,
   return {};
 }
 
+CompilerType TypeSystemSwiftTypeRef::GetBuiltinRawPointerType() {
+  return GetTypeFromMangledTypename(ConstString("$sBpD"));
```

Comment:
It seems fragile to have to know the mangled name for this, is there no way to get this from swift directly?

Reply:
Not without changes in the compiler, and instantiating an ast context (not a problem since we're using this to evaluate expressions, but this function could potentially be called in other places in the future). TypeSystemSwiftTypeRef already deeply depends on understanding swift mangled names already, so this shouldn't be any worse than any code there.